components/results_table/commons/results_table.lua

```diff
@@ -5,3 +5,109 @@
 --
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+---Note: This can be overwritten
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+---Note: This can be overwritten
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
```

Comment:
Just a though, should we put a wiki's opponent libraries in their Module:Info instead of doing this everywhere?

Reply:
good idea but imo out of scope of this pr^^

will open a pr for it later today (adding them to info module)

Reply:
Indeed, definitely out of scope